engine/src/main/java/org/terasology/engine/world/block/BlockPart.java

```diff
@@ -19,10 +18,8 @@
     BOTTOM(Side.BOTTOM),
     CENTER;
 
-    private static final EnumMap<Side, BlockPart> SIDE_MAP;
-    private static final BlockPart[] SIDES;
-    private static final List<BlockPart> HORIZONTAL_SIDES;
-
+    private static final List<BlockPart> HORIZONTAL_SIDES = ImmutableList.of(LEFT, RIGHT, FRONT, BACK);
+    private static final List<BlockPart> SIDES = ImmutableList.of(TOP, LEFT, RIGHT, FRONT, BACK, BOTTOM, CENTER);
```

Comment:
This is just the full set of `values()` - maybe using that makes it bit more obvious that this are just all enum values - basically the cached `.values()` as discussed above.

```suggestion
    /**
     * Cached block parts in an immutable list.
     * See https://dzone.com/articles/memory-hogging-enumvalues-method
     */
    private static final List<BlockPart> SIDES = ImmutableList.of(BlockPart.values());
```